.gitignore

```diff
@@ -7,6 +7,7 @@
 
 .DS_Store
 .idea
+.cxx/
```

Comment:
this should be a separate commit with commit msg, as it's independent of the rest :smile:

Reply:
I wouldn't say it is independent (it's a byproduct of adding the `signal_handler.c` file, at least to my understanding :D) but happy to separate into another commit if you think that's better

Reply:
I see, makes sense.